py/requirements.txt

```diff
@@ -2,4 +2,3 @@ firecloud>=0.16.25
 ipython>=5.7.0
 ipywidgets>=7.5.1
 pandas>=0.25.3
-tensorflow>=2.4.2
```

Comment:
opt: is it worth just dropping the version requirement, or would this still have the same issue?

Reply:
No, it fails https://github.com/deflaux/terra-docker/actions/runs/980750897